website/docs/Capabilities.md

```diff
@@ -240,49 +240,136 @@ When testing on Chrome, WebdriverIO will automatically download the desired brow
+    }
+}
+```
+
+:::caution
+
+If you set a driver binary, the browser won't be downloaded automatically. You'll need to have the corresponding browser already installed or specify a binary path as previously shown.
+
+:::
+
 </TabItem>
 <TabItem value="firefox">
 
-When testing on Firefox, you can let WebdriverIO setup Firefox Nightly for you by providing `latest` as `browserVersion`:
+When testing on Firefox, WebdriverIO will automatically download the desired browser version and driver for you based on the defined `browserVersion`, e.g.:
 
 ```ts
+{
     browserName: 'firefox',
-    browserVersion: 'latest'
+    browserVersion: '119.0a1' // or 'latest'
+}
 ```
 
 If you like to test a manually downloaded version you can provide a binary path to the browser via:
 
 ```ts
+{
     browserName: 'firefox',
     'moz:firefoxOptions': {
-        bin: '/Applications/Firefox\ Nightly.app/Contents/MacOS/firefox'
+        binary: '/Applications/Firefox\ Nightly.app/Contents/MacOS/firefox'
+    }
+}
+```
+
+Additionally, if you like to use a manually downloaded driver, you can provide a binary path to the driver via:
+
+```ts
+{
+    browserName: 'firefox',
+    'wdio:geckodriverOptions': {
+        binary: '/path/to/geckodriver'
     }
+}
 ```
 
+:::caution
+
+If you set a driver binary, the browser won't be downloaded automatically. You'll need to have the corresponding browser already installed or specify a binary path as previously shown.
+
+:::
+
 </TabItem>
 <TabItem value="msedge">
 
 When testing on Microsoft Edge, make sure you have the desired browser version installed on your machine. You can point WebdriverIO to the browser to execute via:
 
 ```ts
+{
     browserName: 'msedge',
     'ms:edgeOptions': {
-        bin: '/Applications/Microsoft\ Edge\ Canary.app/Contents/MacOS/Microsoft\ Edge\ Canary'
+        binary: '/Applications/Microsoft\ Edge\ Canary.app/Contents/MacOS/Microsoft\ Edge\ Canary'
     }
+}
 ```
 
+WebdriverIO will automatically download the desired driver version for you based on the defined `browserVersion`, e.g.:
+
+```ts
+{
+    browserName: 'msedge',
+    browserVersion: '109' // or '109.0.1467.0', 'stable', 'dev', 'canary', 'beta'
+}
+```
+
+Additionally, if you like to use a manually downloaded driver, you can provide a binary path to the driver via:
+
+```ts
+{
+    browserName: 'msedge',
+    'wdio:edgedriverOptions': {
+        binary: '/path/to/msedgedriver'
+    }
+}
+```
+
+:::caution
+
+WebdriverIO won't automatically download Microsoft Edge. Only Chrome and Firefox are supported.
+
+If you set a browser binary and omit the browser version and driver binary, WebdriverIO will auto-determine the driver version to download based on the automatically detected Microsoft Edge installation. In some environments, the detected browser may be different from the specified binary. Therefore, it's recommended to always specify the browser version when specifying a browser binary and omitting the driver binary.
```

Comment:
Ref:

https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/manager.ts#L101-L102

and

https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/utils.ts#L245-L247

and

[webdriverio-community/node-edgedriver@7f2fc5d5b/src/install.ts:25-30](https://github.com/webdriverio-community/node-edgedriver/blob/7f2fc5d5b451f7fbb317a2628d557717ee9a4848/src/install.ts#L25-L30)